applications/hiawatha/hiawatha-10.8.1/config/hiawatha.conf.in

```diff
@@ -41,10 +41,18 @@ StartFile = index.html
 PIDfile = WOKRING_DIR/hiawatha.pid
 WorkDirectory = WOKRING_DIR
 ShowIndex = WEBROOT_DIR/index.html
-UseFastCGI = QFastCGI
+#UseFastCGI = WebHandlerFCGI
+UseToolkit = WebHandlerToolkit
 
 FastCGIserver {
-        FastCGIid = QFastCGI
-        ConnectTo = 127.0.0.1:2005
+        FastCGIid = WebHandlerFCGI
+        ConnectTo = /tmp/web_handler.socket
         SessionTimeout = 30
-}
\ No newline at end of file
+}
+
+UrlToolkit {
```

Comment:
@tranconghau , @nguyenhien1994 , @hvloi 

this is special thing, please try to understand it. :D